doxia-modules/doxia-module-xdoc/src/main/java/org/apache/maven/doxia/module/xdoc/XdocSink.java

```diff
@@ -505,31 +497,6 @@ public void close()
         init();
     }
 
-    /**
-     * Adds a link with an optional target.
-     *
-     * @param name the link name.
-     * @param target the link target, may be null.
-     */
-    public void link( String name, String target )
-    {
-        if ( isHeadFlag() )
-        {
-            return;
-        }
-
-        MutableAttributeSet att = new SinkEventAttributeSet();
-
-        att.addAttribute( Attribute.HREF, HtmlTools.escapeHTML( name ) );
-
-        if ( target != null )
-        {
-            att.addAttribute( Attribute.TARGET, target );
-        }
-
-        writeStartTag( A, att );
-    }
-
```

Comment:
it seems that this link(String, String) method is more than "duplicate code": it's dead code that was introduced into Doxia 1.1 and was never called outside the unit test
then I'm all in to remove this dead code and update the associated unit test, with proper Jira issue renaming

on other changes like VALIGN, that are unrelated to this dead code removal, I'd prefer have a separate commit and review to better understand

and I don't understand why there is a discussion on "source", which seems to be completely an unrelated discussion

Reply:
@hboutemy Done, completely reduced. Attributes are in another ticket. As for the `source`. HTML5 has now a source element which collides with Xdoc's source. That is why I decided not to fiddle with at the moment. Forget about it.